WordPress/Classes/ViewRelated/Reader/Select Interests/ReaderInterestsCollectionViewCell.swift

```diff
@@ -1,6 +1,6 @@
 import UIKit
 
-class ReaderInterestsCollectionViewCell: UICollectionViewCell {
+class ReaderInterestsCollectionViewCell: UICollectionViewCell, NibLoadable {
```

Comment:
```suggestion
class ReaderInterestsCollectionViewCell: UICollectionViewCell, NibReusable {
```

Reply:
Done in: 616998ecec24851a03b947bb70a219a03766ce49